.github/CODEOWNERS

```diff
@@ -3,5 +3,6 @@
 **/giga** @MartinGrignard
 **/shu_bmrc** @lquayle88
 **/utd_ganymede** @edmundmiller @alyssa-ab
+**/utd_juno** @eternal-flame-AD
```

Comment:
Better join nf-core org? (See channel #github-invitations).

Reply:
Done! thanks

Reply:
Hrrm, not sure why it still complains about this. This will need to be fixed before merge I think.

Reply:
Whoever added you to the organisation didn't add you to the specific GitHub team that gives you write permissions - I've done that now

_Please note in the past we've had people reporting they've been signed up to notifications across all nf-core repositories when we add people to the team in this manner. We've not had reports of this recently however, so hopefully it shouldn't happen - but apologies in advance_